python/ENTERPRISE_SUPPORT_PLAN_ENABLED/ENTERPRISE_SUPPORT_PLAN_ENABLED_test.py

```diff
@@ -0,0 +1,206 @@
+# Copyright 2017-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the License is located at
+#
+#        http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+
+import sys
+import unittest
+try:
+    from unittest.mock import MagicMock
+except ImportError:
+    from mock import MagicMock
+import botocore
+
+##############
+# Parameters #
+##############
+
+# Define the default resource to report to Config Rules
+DEFAULT_RESOURCE_TYPE = 'AWS::::Account'
+
+#############
+# Main Code #
+#############
+
+CONFIG_CLIENT_MOCK = MagicMock()
+STS_CLIENT_MOCK = MagicMock()
+SUPPORT_CLIENT_MOCK = MagicMock()
+
+class Boto3Mock():
+    @staticmethod
+    def client(client_name, *args, **kwargs):
+        if client_name == 'config':
+            return CONFIG_CLIENT_MOCK
+        if client_name == 'sts':
+            return STS_CLIENT_MOCK
+        if client_name == 'support':
+            return SUPPORT_CLIENT_MOCK
+        raise Exception("Attempting to create an unknown client")
+
+sys.modules['boto3'] = Boto3Mock()
+
+RULE = __import__('ENTERPRISE_SUPPORT_PLAN_ENABLED')
+
+class NonCompliantResourceTest(unittest.TestCase):
+    def test_scenario_1_baisc_support_non_compliant(self):
+        SUPPORT_CLIENT_MOCK.describe_severity_levels = MagicMock(
+            side_effect=botocore.exceptions.ClientError(
+                {'Error': {'Code': 'SubscriptionRequiredException', 'Message': 'unknown-message'}},
+                'operation'
+                )
+            )
+        lambda_result = RULE.lambda_handler(build_lambda_scheduled_event(), {})
+        assert_successful_evaluation(
+            self,
+            lambda_result,
+            [build_expected_response(
+                'NON_COMPLIANT',
+                '123456789012',
+                DEFAULT_RESOURCE_TYPE,
+                'The AWS Enterprise Support Plan is not enabled for this AWS Account.'
+                )]
+            )
+    def test_scenario_2_bussiness_support_non_compliant(self):
+        SUPPORT_CLIENT_MOCK.describe_severity_levels = MagicMock(
+            return_value={'severityLevels': [{'code': 'urgent', 'name': 'Urgent'}]}
+            )
+        lambda_result = RULE.lambda_handler(build_lambda_scheduled_event(), {})
+        assert_successful_evaluation(
+            self,
+            lambda_result,
+            [build_expected_response(
+                'NON_COMPLIANT',
+                '123456789012',
+                DEFAULT_RESOURCE_TYPE,
```

Comment:
No need to write DEFAULT_RESOURCE_TYPE here. It is passed by default by the function.